src/Database/Validator/Datetime.php

```diff
@@ -37,7 +45,13 @@ public function isValid($value): bool
         }
 
         try {
-            new \DateTime($value);
+            $date = new \DateTime($value);
+            $now = new \DateTime();
+
+            if ($this->allowOldDates === false && $date < $now) {
+                $this->message = 'Date is in the past';
```

Comment:
Let's update this too:

```suggestion
                $this->message = 'Date must be in the future';
```

Reply:
done this